models/shortcode.go

```diff
@@ -1,11 +1,15 @@
 package models
 
-import "gorm.io/gorm"
+import (
+	"gorm.io/gorm"
+	"time"
+)
 
 type Shortcode struct {
 	gorm.Model
-	Code    string
-	Request string
-	UserID  uint
-	User    User
+	Code      string
+	Request   string
+	UserID    uint
+	User      User
+	CreatedOn time.Time `gorm:"autoCreateTime"`
```

Comment:
Why not use the default `CreatedAt` of the `gorm.Model` for this?

Reply:
just though on having 1 on 1 mapping with graphql as wasn't able to figure out conviction for timing for this specific project, you're right as in general it's CreatedAt or UpdatedAt.

can update if you want.

Reply:
Just checked docs, my bad i haven't worked with gorm before. i'll update it right away